modules/cnf-sample-single-numa-policy-from-pp.adoc

```diff
@@ -32,5 +32,5 @@ spec:
     realTime: true
 ----
 
-<1> This should match the `MachineConfigPool` that you want to configure the NUMA Resources Operator on. For example, you might have created a `MachineConfigPool` named `worker-cnf` that designates a set of nodes that run telecommunications workloads.
+<1> This should match the `MachineConfigPool` that you want to configure the NUMA Resources Operator on. For example, you might have created a `MachineConfigPool` named `worker-cnf` that designates a set of nodes that run telecommunications workloads. It should match the `machineConfigPoolSelector` in the `NUMAResourcesOperator` CR configured later in "Creating the NUMAResourcesOperator custom resource".
```

Comment:
same as my comment above, there's no actual meaning to the selector in HyperShift.
It's all done via the `nodePool` object.